platformcomponents/desktop/text-input.json

```diff
@@ -0,0 +1,59 @@
+{
+  "textinput": {
+    "comment": "Used for all text inputs and their states",
+    "figma": "https://www.figma.com/file/NaNrfXjygZtRgMfHAFHjsp/Components---Windows%2BWeb?node-id=4429%3A871",
+    "placeholder-text": "@theme-text-secondary-normal",
+    "#normal": {
+      "background": "@theme-background-solid-primary-normal",
+      "text": "@theme-text-primary-normal",
+      "border": "@theme-outline-input-normal"
+    },
+    "#pressed": {
+      "background": "@theme-background-secondary-active",
+      "text": "@theme-text-primary-normal",
```

Comment:
The only difference between this newly-added file and `components/text-input.json` is changing the `pressed` text colour from secondary to primary
<img width="631" alt="Screenshot 2022-02-16 at 15 08 07" src="https://user-images.githubusercontent.com/97953811/154293757-335ec59d-1a5b-4064-a2ca-d072e5ca9acd.png">